src/server/worldserver/worldserver.conf.dist

```diff
@@ -2843,6 +2843,30 @@ AllowTrackBothResources = 0
 
 PlayerStart.AllReputation = 0
 
+#
+#    XP.Boost.Daymask
+#        Description: Enables experience boost during the defined days. This field is a bitmask.
+#
+#        Default:     0 - (Disabled)
+#                     1 - (Sunday)
```

Comment:
does this depend on the OS weekday definition ? (for example Egypt has a different weekend)

Reply:
The code uses `now.tm_wday` and tm_wday is defined as `days since Sunday – [0, 6]` as seen here https://en.cppreference.com/w/c/chrono/tm
I tested locally to enable monday only yesterday (which is the first day of the week here and in my system) and the system works as documented in the config (monday = 2).
From what I see it is not related to OS or locale etc. according to my tests or the documentation for `tm` struct.

Reply:
Thanks for the info 👍